CMakeLists.txt

```diff
@@ -1458,6 +1488,15 @@ ocv_build_features_string(parallel_status EXCLUSIVE
   ELSE "none")
 status("")
 status("  Parallel framework:" "${parallel_status}")
+if (OPENCV_DISABLE_THREAD_SUPPORT)
+  status("" "Multi thread code explicitly disabled with OPENCV_DISABLE_THREAD_SUPPORT.")
+  if(HAVE_PTHREADS_PF OR HAVE_HPX OR HAVE_OPENMP OR HAVE_GCD OR HAVE_CONCURRENCY)
+    status("" "Warning: not all parallel frameworks have been disabled (using ${parallel_status}).")
```

Comment:
Why are these warnings should go into `getBuildInformation()`?

Reply:
Yeah, they are here mostly because I was using them for debugging when I was developing this patch. I am happy to remove them, or to move them to a more appropriate place. Let me know what you prefer.

Reply:
Prefer to use CMake's `message(WARNING ..)`

Reply:
In the same place? Or should I move it to a place where all warnings are?

Reply:
I'll mark this as done, the warnings are in the same place where the status messages were. Let me know if you want me to move them somewhere else.